pkg/kademlia/utils_test.go

```diff
@@ -26,7 +26,7 @@ func TestSortByXOR(t *testing.T) {
 }
 
 func BenchmarkSortByXOR(b *testing.B) {
-	nodes := []storj.NodeID{}
+	var nodes []storj.NodeID
```

Comment:
nit: since k=1000 is known, `nodes := make([]storj.NodeID, 1000)` would be better